code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -68,6 +68,8 @@
 	if(..())
 		return 1
 
+	if(pulling && istype(pulling, /mob/living/carbon/human))
+		stop_pulling()

```

Comment:
oh no its why i post it here. Anyone have a better solution for this or whats going on with xeno organs shit?